### ISSUE TITLE
feat(patch): change Twitter blue users icon 

### ISSUE THREAD

**Issue by castdrian** [revanced/revanced-patches#1022]
### Application

Twitter

### Issue

Essentially replicate what this repo does for chromium https://github.com/wesbos/who-blue

### Patch

Pretty much a replica of the extension or something in that area

### Motivation

Elon Musk is a twat and we all love to spite him

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by DananaBanana** [comment]
Vouch, I would opt for a design more like this: https://github.com/wseagar/eight-dollars, however I'd maybe remove the text and make it icon-only.

**Comment by MoonlightCapital** [comment]
Second, but I'd like the sham checkmark to be replaced by the nerd face (so basically this: https://twitter.com/shadowbIood/status/1590457493750415361)

If there's so much disagreement over what the new icon should be let the user choose.

**Comment by dylangerdaly** [comment]
I'm working on a patch that'll just not render people that have Twitter Blue as verified.

I use Twitter daily and don't want to see a verified check next to people that paid for it.

Currently, you can differentiate people that paid and people that didn't

**Comment by castdrian** [comment]
Yeah but the fun of it would be to have them stand out in a humorous way, so maybe the ability to add an svg from storage to replace it would be fine, like just an extra setting in the menu that controls the behavior of the patch, ie don't show blue checkmark or replace them with an svg

**Comment by dylangerdaly** [comment]
If you'd like to see all of the feature prefs currently available for Android: https://paste.sr.ht/~dylanger/ec37db39a14528eb9b3de8d59e99cbde112039be

Currently `subscriptions_android_is_blue_verified_field_enabled` is what enables or disables blue checkmarks.

**Comment by dnestico** [comment]
+1
